serviceapis/twitterapi.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+'''
+Copyright © 2017, ACM@UIUC
+FoodButton is open source software, released under the University of
+Illinois/NCSA Open Source License.  You should have received a copy of
+this license in a file with the distribution.
+'''
+
+from .serviceapi import ServiceAPI
+from datetime import datetime
+import tweepy
+from PIL import Image
+
+class TwitterAPI(ServiceAPI):
+    '''
+    Implements the ServiceAPI interface for Twitter
+    '''
+    def __init__(self, cfg):
+        auth = tweepy.OAuthHandler(cfg['consumer_key'], cfg['consumer_secret'])
+        auth.set_access_token(cfg['access_token'], cfg['access_token_secret'])
+        self.api = tweepy.API(auth)
+         
+    def post(self, img, text):
+        try:
+            filename = "/tmp/food_on_table_{}.png".format(datetime.now())
```

Comment:
I'd consider using the python [TemporaryFile](https://docs.python.org/2/library/tempfile.html#tempfile.TemporaryFile) interface, as it automatically cleans up the tempfile after use.

Reply:
Do you know a way to use TemporaryFile with PIL? When the camera is added I suspect something like opencv will be used to extract the image from the camera so we would need a workflow compatible with that (is why we are using PIL for the dummy function rn)

Reply:
Cool